_episodes/51.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: episode
+title: "51: Trino cools off with PopSQL"
+date: 2023-07-06
+tags: trino popsql query editor collaboration
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Intro"
+  desc: "An intro to PopSQL"
+  time: 0
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Technical Content at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* Jake Peterson, Head of Customer Success at [PopSQL](https://popsql.com/)
+* Matt Peveler, Software Engineer at [PopSQL](https://popsql.com/)
+
+## Releases 423-427
+
+To be filled out shortly...
+
+## Concept of the episode: PopSQL
+
+It may be familiar to some of our viewers to describe an environment where
+key queries and dashboards are buried in someone's personal workspace, and you
+have to go ask them directly every time you want to check on a tool's health.
```

Comment:
what do you mean by a tools health ... would it be better to use some business metric